src/main/java/frc/robot/subsystems/swervedrive/SwerveSubsystem.java

```diff
@@ -0,0 +1,390 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
```

Comment:
is this wpilib/FIRST code? or did you write it? if it is your code, you can get rid of the copyright. if wpilib, you should really just use this from the library instead of copying it in